pages/cfv.md

```diff
@@ -1,6 +1,9 @@
-<h1> Calling for venues </h1>
+<h1> Call for venues </h1>
```

Comment:
Should we have this as Continuous Call same as for the speakers page?

Reply:
Yes, I agree with this for coherency 👍 Just did it